sdcm/utils/gce_utils.py

```diff
@@ -463,10 +463,10 @@ def create_instance(  # pylint: disable=too-many-arguments,too-many-locals,too-m
         instance.guest_accelerators = accelerators
 
     instance.scheduling = compute_v1.Scheduling()
-    instance.scheduling.on_host_maintenance = "TERMINATE"
 
     if spot:
         # Set the Spot VM setting
+        instance.scheduling.on_host_maintenance = "TERMINATE"
```

Comment:
this gonna break the new z3, where we need this regardless of spot or not

Reply:
but currently anything else seems to be broken...

Reply:
> but currently anything else seems to be broken...

So let's revert and fix it when interducing it again